lib/std/os/uefi/protocols/device_path_protocol.zig

```diff
@@ -49,14 +52,20 @@ pub const DevicePathProtocol = packed struct {
 
         // Pointer to the copy of the end node of the current chain, which is - 4 from the buffer
         // as the end node itself is 4 bytes (type: u8 + subtype: u8 + length: u16).
-        var new = @ptrCast(*MediaDevicePath.FilePathDevicePath, buf.ptr + path_size - 4);
+        var new = @ptrCast(
+            *MediaDevicePath.FilePathDevicePath,
+            @alignCast(4, buf.ptr + path_size - 4),
+        );
 
         new.type = .Media;
         new.subtype = .FilePath;
         new.length = @sizeOf(MediaDevicePath.FilePathDevicePath) + 2 * (@intCast(u16, path.len) + 1);
 
         // The same as new.getPath(), but not const as we're filling it in.
-        var ptr = @ptrCast([*:0]u16, @alignCast(2, @ptrCast([*]u8, new)) + @sizeOf(MediaDevicePath.FilePathDevicePath));
+        var ptr = @ptrCast(
+            [*:0]u16,
+            @alignCast(2, @ptrCast([*]u8, new) + @sizeOf(MediaDevicePath.FilePathDevicePath)),
```

Comment:
Probably don't want the `@alignCast`s here now

Reply:
That needs to be aligned because u16 is align(2)

Reply:
Oh sorry I meant the one after that with `DevicePathProtocol`, I believe the u16 strings are aligned. `FilePathDevicePath` shouldn't be aligned either I think, given it's a subtype of `DevicePathProtocol`.